### PR TITLE
add Debian

### PR DESCRIPTION
**Pull request by aba-rechsteiner**

### CODE REVIEW

**Comment by johnlawerance** [comment]
@aba-rechsteiner are you looking to add install specifics for debian? If so I'd be happy to merge and push out a release. I haven't touched this project for quite a while but happy to get it in master.

**Comment by aba-rechsteiner** [comment]
yes we use the module for fisheye under Debian GNU/Linux 10 (buster)

**Comment by johnlawerance** [comment]
Thanks @aba-rechsteiner -- This block in this manifest is for installing java if `$crucible::install_java == true` and the Debian block added won't do that. Are you intending on adding that?

File: manifests/install.pp [hunk 1]
```diff
@@ -31,6 +31,9 @@
         }
       }
 
+      'Debian':{
```
Comment (aba-rechsteiner) [inline manifests/install.pp:34]: @wolfaba maybe we should add a
`include java`

Comment (wolfaba) [reply manifests/install.pp:34]: @johnlawerance oh sorry, I have missed, this is only for Java and only if `install_java==true`.

@aba-rechsteiner OK, for Debian block there should be any `include java`. But I would like install java myself, therefore I don't need it in fact. We can disable `install_java` in our installation, becase we install java from our profile module.

AFAIK Jira and Confluence modules neither install java themself, so for me it's OK, if crucible module will not install java. But I understand for someone can be useful to install java for crucible in one config.

Comment (aba-rechsteiner) [reply manifests/install.pp:34]: Okay, then I'll close this PR

**Pull request closed without merging**